docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-overlay.mdx

```diff
@@ -98,3 +102,71 @@ fun SelectedMessageOverlay(
 * `messageOptions`: Allows you to customize which message options are shown in the overlay. You can use `defaultMessageOptions()` to get the default actions that we expose in our SDK.
 
 If you want to customize these options or reactions, you can set up custom reactions, using the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx)'s `reactionTypes` parameter.
+
+By default `SelectedMessageMenu` is made to resemble a bottom sheet, however it can be stylised in order to look like a completely different component.
```

Comment:
I'd like to break this into subsections because I think this step by step logic is better than simply listing parameters and giving a short description.

What I don't like is how it all looks bunched together so if anyone has some appropriate markdown-fu suggestions I am open to them. :D

Reply:
By default `SelectedMessageMenu` is made to resemble a bottom sheet, however it can be stylised in order to look like a completely different component.

->

By default the `SelectedMessageMenu` looks like a bottom sheet, however you can customize it to look like a completely different component, such as a dialog.

Reply:
Listing parameters shouldn't be removed - it's our doc structure and gives our users more info on what we expose.

As for the step-by-step approach - that's good, and can follow the list of params, but we don't want lengthy doc pages that descibe the basic options of a component.

In this case it's okay, but for anyhting more than that, we should instead write a guide.